scss/mixins/_buttons.scss

```diff
@@ -39,8 +39,8 @@
     &:focus,
     &.focus {
       color: $color;
-      background-color: darken($background, 17%);
-          border-color: darken($border, 25%);
+      background-color: darken($background, ($btn-darken + 7));
```

Comment:
Presumably `$btn-darken + 7` and `$btn-darken + 13` should be variables instead of having hardcoded magic numbers.